integration

```diff
@@ -46,6 +46,7 @@
   "amosyuen/ha-tplink-deco",
   "anarion80/sodexo_dla_ciebie",
   "And3rsL/VisonicAlarm-for-Hassio",
+  "andersevenrud/ha-nexa-bridge-x",
```

Comment:
<https://hacs.xyz/docs/publish/include>
> Custom integrations that exist to alpha/beta test core integrations will not be accepted, you can still use it as a [custom repository](https://hacs.xyz/docs/faq/custom_repositories)

<https://github.com/andersevenrud/ha-nexa-bridge-x/issues/7>

Reply:
I created that issue mostly for investigative purposes and haven't got any plans to actually publish this as a core integration.

Maybe you could reconsider and re-open this PR @ludeeus ?